src/app/pages/common/ix/components/ix-user-combobox/ix-user-combobox.component.scss

```diff
@@ -58,3 +58,7 @@
 .ix-selected {
   background: var(--primary) !important;
 }
+
+.ix-unselected {
```

Comment:
I thinking of reusing the base styles for custom implementation of the combobox. To minimize the chance of error and simplify maintenance. Maybe it would make sense to  `@import ix-combobox.component.scss` in `ix-user-combobox.component.scss`?